foreman/data_refinery_foreman/surveyor/harmony.py

```diff
@@ -0,0 +1,699 @@
+                    harmonized_samples[title]['sex'] = "female"
+                    break
+                else:
+                    harmonized_samples[title]['sex'] = "male"
+                    break
+
+    ##
+    # Age!
+    ##
+    age_fields = [    
+                    'age',
+                    'patient age',
+                    'age of patient',
+                    'age (years)',
+                    'age at diagnosis',
+                    'age at diagnosis years',
+                    'characteristic [age]',
+                    'characteristics [age]',
+                ]
+    age_fields = add_variants(age_fields)
+
+    for sample in original_samples:
+        title = sample['title']
+        for key, value in sample.copy().items():
+            lower_key = key.lower().strip()
+            if lower_key in age_fields:
+                try:
+                    harmonized_samples[title]['age'] = float(value)
+                except ValueError:
+                    try:
+                        harmonized_samples[title]['age'] = float(value.split(' ')[0])
+                    except ValueError:
+                    	# This is probably something weird, like a '.'
+                        continue
+                break
+
+    ##
+    # Organ Parts!
+    # Cell Type and Organ Type are different but grouped,
+    # See: https://github.com/AlexsLemonade/refinebio/issues/165#issuecomment-376684079
+    ##
+    part_fields = [
+                    # AE
+                    'organism part', 
+                    'cell type', 
+                    'tissue', 
+                    'tissue type', 
+                    'tissue source', 
+                    'tissue origin', 
+                    'source tissue', 
+                    'tissue subtype', 
+                    'tissue/cell type', 
+                    'tissue region', 
+                    'tissue compartment', 
+                    'tissues', 
+                    'tissue of origin', 
+                    'tissue-type', 
+                    'tissue harvested', 
+                    'cell/tissue type', 
+                    'tissue subregion', 
+                    'organ',
+                    'characteristic [organism part]',
+                    'characteristics [organism part]',
+
+                    # SRA
+                    'cell_type'
+                    'organismpart',
+
+                    # GEO
+                    'isolation source',
+                    'tissue sampled',
+                    'cell description'
+
+                ]
+    part_fields = add_variants(part_fields)
+    for sample in original_samples:
+        title = sample['title']
+        for key, value in sample.copy().items():
+            lower_key = key.lower().strip() 
+            if lower_key in part_fields:
+                harmonized_samples[title]['part'] = value.lower().strip()
+                break
+
+    ##
+    # Genotype!
+    ##
+    genotype_fields = [
```

Comment:
I am a bit uncomfortable with lumping these together given the way we now intend to use the harmonized data.

From my original comment (https://github.com/AlexsLemonade/refinebio/issues/165#issuecomment-376684079): 

> For filtering only (i.e., we're mostly interested in the presence or absence of metadata attached to a sample), I think it's appropriate to lump together the following categories: genotype, genetic background, variation, strain or line, background strain, cultivar and ecotype. This basically would indicate that there is some information about the genetic background (in a broad sense) that is associated with an individual sample. These are not the exact same information (multiple cultivars could potentially have the same genetic background and I bet people use "genotype" and "genetic background" a wide range of different ways depending on their individual experiments)

Reply:
Genetic information is better than genotype (since genotype is likely to be represented by both a strain background + variations from that such as deletions).